README.md

```diff
@@ -156,16 +156,15 @@ Gluten is licensed under [Apache 2.0 license](https://www.apache.org/licenses/LI
 
 # 7 Contact
 
-Gluten was initiated by Intel and Kyligence in 2022. Several companies such as Intel, Kyligence, BIGO, Meituan, Alibaba Cloud, NetEase, Baidu, Microsoft and others, are actively participating in the development of Gluten. If you are interested in Gluten project, please contact below email address for further discussion.
+Gluten was initiated by Intel and Kyligence in 2022. Several other companies are also actively participating in the development, such as BIGO, Meituan, Alibaba Cloud, NetEase, Baidu, Microsoft, etc. If you are interested in Gluten project, please contact and subscribe below email address for further discussion.
 
-binwei.yang@intel.com; weiting.chen@intel.com;
-chang.chen@kyligence.io; zhichao.zhang@kyligence.io; neng.liu@kyligence.io;
-zuochunwei@meituan.com;yangchuan.zy@alibaba-inc.com;xiyu.zk@alibaba-inc.com;joey.ljy@alibaba-inc.com
+dev@gluten.apache.org for community activity
```

Comment:
Since we have the concat email here, do we sill need the `concat-us.md`?

Reply:
I think we can remove that doc. Will remove that.